test-requirements-py2.txt

```diff
@@ -1,6 +1,6 @@
 jinja2
 pytest
 mock
-rpm-py-installer
+rpm-py-installer < 4.16
```

Comment:
If you need this then please do it in its own PR with the reasoning explained.

Reply:
I accidentally included it in this commit